src/components/ComputeModal.js

```diff
@@ -190,6 +190,8 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     hasGpu: false,
     gpuType: defaultGpuType,
     numGpus: defaultNumGpus,
+    autopauseEnabled: defaultAutoPause,
+    autopauseThreshold: defaultAutoPauseLength,
```

Comment:
I see that we use both `threshold` and `length`. Can we pick one and use that so there's less terminology? I'd vote for `threshold` but it's your call :)

Reply:
Yes agreed! Making the change for that default value now.